lib/remote/actionshandler.cpp

```diff
@@ -100,11 +100,33 @@ bool ActionsHandler::HandleRequest(
 	}
 
 	int statusCode = 500;
+	Array statusCodes;
```

Comment:
Array? Aren't there better data structures for unique values?

Reply:
And they are? In any case I have not found a better one.

Reply:
`std::set<int>`

Reply:
But I wanted to use our data structure and not from `std`.

Reply:
What for?

Reply:
Because they are there. What were they built for, if they can be used?

Reply:
For their use cases. This one isn't a such.